### DISCUSSION TITLE
Query: config.uae name matching when using floppy images.

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1291] [Ideas]
The following procedure works;

-launch Amiberry-> GUI -> Quickstart -> select whdload.lha file -> Start ...you decide you want this title to launch fullscreen..
-F12 -> Display -> set Screen mode: to Fullscreen -> Configurations -> Name of .lha archive is pre-filled, so click Save button...

The next time this specific whdload.lha file is loaded/started by amiberry, it will use the created config.uae file instead of using the XML file settings....all good here.

Now that amiberry also pre-fills the `Name` field with the name of the floppy image in DF0: when creating/saving a config.uae file using the procedure outline above, I would have expected amiberry to find that named config.uae file, the next time the same named floppy image is loaded into DF0: and the emulation started.

Currently however, it seems amiberry doesn't check the conf/ directory to see if a config.uae file exists that matches the floppy image name in DF0:, and merely starts the emulation as it would in Quickstart with no config.uae support.

I'm not entirely sure if this is a bug, or just something we didn't think about at the time ;) Either way, I believe amiberry should check if an appropriately named config.uae file exists, for the floppy image loaded into DF0: ?

TIA

**Comment by midwan** [comment]
I'm not sure that's the best way to go.

If you load up a floppy image from the command line, it will indeed check if there's a config file for it and load it automatically. 

But when you're in the GUI, auto-loading a config based on a disk you insert feels wrong, and a source for problems. What if you only want the disk inserted with no config changed? You might want to test something with another setup. 

Why not just load the config directly yourself, instead of the disk? It will contain the inserted disk as well as everything else, and you're already in the GUI, so there's no overhead.

**Comment by giantclambake** [comment]
You're quite right, my mistake here ~ I totally forgot to check cmdline invocation (where it works as expected)... apologies.
